src/dune_file.ml

```diff
@@ -1035,6 +1035,181 @@ module Install_conf = struct
 end
 
 module Executables = struct
+  module Names : sig
+    type t
+
+    val names : t -> (Loc.t * string) list
+
+    val has_public_name : t -> bool
+
+    val make
+      :  multi:bool
+      -> stanza:string
+      -> allow_omit_names_version:Syntax.Version.t
+      -> (t, fields) Dune_lang.Decoder.parser
+
+    val install_conf
+      : t
+      -> ext:string
+      -> (String_with_vars.t Install_conf.t, string) Result.t option
+
+    val best_package_loc : t -> Loc.t
```

Comment:
I couldn't guess the semantic of this function without looking at the implementation. I suggest to call it `loc_of_package_field`

Reply:
done